### PR TITLE
Added Tilda & Synapse

### PR DESCRIPTION
**Pull request by kgleason**
I added in support for a couple of tools that I use in Linux frequently.

### CODE REVIEW

**Comment by lra** [comment]
https://github.com/lra/mackup/blob/master/CONTRIBUTING.md

**Comment by kgleason** [comment]
Apologies. I'll fix and resubmit.

File: mackup/applications/synapse.cfg [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[application]
+# http://synapse.zeitgeist-project.com/wiki/index.php?title=Main_Page
+name = Synapse
+
+[configuration_files]
+.config/synapse
```
Comment (vitorgalvao) [inline mackup/applications/synapse.cfg]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

File: mackup/applications/tilda.cfg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[application]
+# https://github.com/lanoxx/tilda
+name = Tilda
+
+[configuration_files]
+.config/tilda
```
Comment (vitorgalvao) [inline mackup/applications/tilda.cfg]: Same here.

**Pull request closed without merging**